api/src/opentelemetry/trace/span_id.cc

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019, OpenTelemetry Authors
```

Comment:
Re. file naming convention - if the only class in that module is SpanId , should we name the file **SpanId[.cpp|.hpp]** or **SpanId[.cc|.hh]** ? .cpp looks a bit better to my eye, others may have a different opinion :)